.github/workflows/upload-pypi-source.yml

```diff
@@ -0,0 +1,48 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  upload_source_dist:
+    name: Upload source distribution
+    runs-on: [macos-latest]
+
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        fetch-depth: 0
+        # we need fetch-depth 0 so setuptools_scm can resolve tags
+    - uses: actions/setup-python@v1
+      name: Install Python
+      with:
+        python-version: '3.7'
+
+    - name: Build source distribution
+      run: |
+        pip install pep517
+        python -m pep517.build --source --out-dir dist .
```

Comment:
So the way to fix the bootstrapping problem is to just call "python setup.py" (or install scons and pytoml, and run python -c "import enscons; enscons.setup.setup()". That will package enscons with itself building both the sdist and the wheel.